### ISSUE TITLE
Maximum  Circular Subarray Sum.[C++][GSSOC'21]

### ISSUE THREAD

**Issue by anishsingh42** [HarshCasper/NeoAlgo#4138]
## 💥 Proposal

Maximum Circular Subarray sum.
The word circular means, the end of the array is connected to the beginning element of the array. Finding the maximum sum of consecutive numbers. This is the best problem based on the implementation of Kadane's Algorithm.
Preferred Language C++ 
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?
Yes

**Comment by ankitaggarwal23** [comment]
@anishsingh42 Already someone did this problem in C++ section. Kindly don't open such issue always check before opening a new issue.
